extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -174,6 +185,48 @@ export class PlatformStore {
     return this.grantStatusPromise_;
   }
 
+  /**
+   * Checks and saves the entitlement for grant status
+   * @param {!Entitlement} entitlement
+   * @private
+   */
+  saveGrantEntitlement_(entitlement) {
+    if ((!this.grantStatusEntitlement_) || (this.grantStatusEntitlement_
```

Comment:
Please comment the rules above - the `if` is readable, but still complicated enough to warrant additional explanation.